### DISCUSSION TITLE
Etherscan Api not working.

### DISCUSSION THREAD

**Discussion by poohGeorge** [community/community#122232] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

I am trying to use etherscan api key to register web3 optimism explorer.
But it throws time out of registration error.

**Comment by dragonwarrior9873** [comment, marked as answer]
That's because yours is not suitable for etherscan. Optimism is differenet from Etherem mainnnet.
 Please navigate https://etherscan.api/register and get your own scan key.
